src/librustdoc/clean/types.rs

```diff
@@ -1839,10 +1839,10 @@ impl PrimitiveType {
                 //
                 // Either manually update this arrayvec at this point
                 // or start with a more complex refactoring.
-                Tuple => [TupleSimplifiedType(2), TupleSimplifiedType(3)].into(),
+                Tuple => [TupleSimplifiedType(1), TupleSimplifiedType(2), TupleSimplifiedType(3)].into(),
```

Comment:
I've poked around this part of the code in an IDE and I can't figure out what it does. Would you mind briefly explaining it to me, along with why it needed to change?

Reply:
Thanks!